sdk/openai/Azure.AI.OpenAI/src/Custom/AzureChatExtensions/PineconeChatExtensionConfiguration.cs

```diff
@@ -77,6 +77,5 @@ public PineconeFieldMappingOptions FieldMappingOptions
     public OnYourDataVectorizationSource VectorizationSource
     {
         get => Parameters.EmbeddingDependency;
-        set => Parameters.EmbeddingDependency = value;
```

Comment:
is that because this property is read-only in the original spec?

Reply:
I think it is because the model usage is changed, it is not roundtrip, so there is no `setter`, we need to update the customized code accordingly.